var/spack/repos/builtin/packages/py-httplib2/package.py

```diff
@@ -12,7 +12,11 @@ class PyHttplib2(PythonPackage):
     homepage = "https://github.com/httplib2/httplib2"
     pypi = "httplib2/httplib2-0.13.1.tar.gz"
 
+    version("0.22.0", sha256="d7a10bc5ef5ab08322488bde8c726eeee5c8618723fdb399597ec58f3d82df81")
     version("0.18.0", sha256="b0e1f3ed76c97380fe2485bc47f25235453b40ef33ca5921bb2897e257a49c4c")
     version("0.13.1", sha256="6901c8c0ffcf721f9ce270ad86da37bc2b4d32b8802d4a9cec38274898a64044")
 
+    depends_on("py-setuptools@40.8.0:", when="@0.18.1:", type="build")
     depends_on("py-setuptools", type="build")
+
+    depends_on("py-pyparsing@2.4.2:2,3.0.4:3", when="@0.19:", type=("build", "run"))
```

Comment:
For these kinds of things I usually do this nowadays:
```suggestion
    depends_on("py-pyparsing@2.4.2:3", when="@0.19:", type=("build", "run"))
    conflicts("py-pyparsing@3.0.1:3.0.3", when="@0.19:")
```
But both options work.